.github/workflows/check-shell.yml

```
@@ -2,6 +2,9 @@ name: "Check shell"
 
 on:
   pull_request_target:
+    paths:
+      - 'shell.nix'
+      - './ci/**'
```

```suggestion
      - 'ci/**'
```

All examples here do not include `./`
https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet

---

As long as it works I don't think we should care about such nits :)

---

`./ci/**` did not work in practice.

I opened 2 PRs in my own nixpkgs fork.

1) Edit `shell.nix`: https://github.com/azuwis/nixpkgs/pull/1

2) Edit `ci/default.nix` https://github.com/azuwis/nixpkgs/pull/2

You can see in https://github.com/azuwis/nixpkgs/actions/workflows/check-shell.yml that only https://github.com/azuwis/nixpkgs/pull/1 trigger the `check-shell` workflow.

---

https://github.com/NixOS/nixpkgs/pull/360283

---

Oh damn, good catch, I wouldn't have expected that!